docs/src/increase-observability/integrate-observability/new-relic/php.md

```diff
@@ -11,15 +11,15 @@ Sign up at https://newrelic.com and get your [license key](https://docs.newrelic
 Add your New Relic license key as an environment level variable:
 
 ```bash
-platform variable:create --level environment --environment '<your-environment>' --visible-build false --inheritable false php:newrelic.license --value '<your-new-relic-license-key>'
+platform variable:create --level environment --environment '<your-environment>' --visible-build false --inheritable false --json false --sensitive true --enabled true --visible-runtime true php:newrelic.license --value '<your-new-relic-license-key>'
 ```
 
 ## Give your application a name
 
 Add a new environment level variable to give your application a recognizable name:
 
 ```bash
-platform variable:create --level environment --environment '<your-environment>' --visible-build false --inheritable false php:newrelic.appname --value '<your-application-name>'
+platform variable:create --level environment --environment '<your-environment>' --visible-build false --inheritable false --json false --sensitive true --enabled true --visible-runtime true php:newrelic.appname --value '<your-application-name>'
```

Comment:
:hammer_and_wrench: As above.

```suggestion
platform variable:create --level environment --environment '<your-environment>' --visible-build false --inheritable false --sensitive true php:newrelic.appname --value '<your-application-name>'
```

Reply:
Hi @CollierCZ, sorry but I have a bit of a preference here. Per my understanding, the user should be able to copy and run the command without any additional interactions. But again I am very open to the conversation :) Maybe adding line breaks for these commands would be some sort of compromise between our perspectives?